src/opinions/validateOpinion.ts

```diff
@@ -0,0 +1,12 @@
+import Joi from "joi";
+
+export default function validateOpinion(req: string) {
+  const schema = Joi.object({
+    content: Joi.string().min(2).max(500).required(),
+    userId: Joi.string().required(),
+    mentorId: Joi.string().required(),
+    stars: Joi.number().min(1).max(5)

```

Comment:
https://stackoverflow.com/questions/49818939/what-is-the-function-to-define-the-minimum-and-maximum-length-of-a-integer-varia

![obraz](https://user-images.githubusercontent.com/56917808/159156201-7b342dd0-3923-4bde-9015-9ae988bacedd.png)